pkg/audit/cursor_test.go

```diff
@@ -112,9 +112,9 @@ func TestDeleteExpired(t *testing.T) {
 		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
 		//populate metainfo with 10 expired pointers of test data
-		_, cursor, metainfo := populateTestData(t, planet, time.Now().Add(-time.Second*1))
+		_, cursor, metainfoService := populateTestData(t, planet, time.Now().Add(-time.Second*1))
```

Comment:
`svc` always sounded to me a good abbreviation for `service`.

Not asking for a change here, just commenting to find out what others think about.

Reply:
metainfoSvc sounds good to me